test/platform/lists.relationshipTypes.xsd

```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In general, could you please shorten your `xsd`, ... files so that the library isn't bloated with overloaded files just for testing purposes. I guess just a few child nodes per node will be fine - a whole list of countries or values which are basically all the same is not necessary for testing imho.